vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -160,8 +172,7 @@ export WORKERS_KUBEADM_EXTRA_CONFIG="
         - chmod a+x kubeadm kubelet kubectl
         - mv kubeadm kubelet kubectl /usr/local/bin/
         - mkdir -p /etc/systemd/system/kubelet.service.d
-        - curl -sSL \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service\" | sed \"s:/usr/bin:/usr/local/bin:g\" > /etc/systemd/system/kubelet.service
-        - curl -sSL \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf\" | sed \"s:/usr/bin:/usr/local/bin:g\" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+        - retrieve.kubeadm.kubelet.configs.sh
```

Comment:
this file will not exist in the workers, you need to define it here too

Reply:
added